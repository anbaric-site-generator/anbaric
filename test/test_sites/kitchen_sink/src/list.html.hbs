<h1>Top-level</h1>
<ul>
    {{#each site.pages}}
        <li>{{ url }}</li>
    {{/each}}
</ul>

<h1>Subdirectory</h1>
<ul>
    {{#each (lookup site.pages "subdir")}}
        <li>{{ url }}</li>
    {{/each}}
</ul>

<h1>Sub-subdirectory</h1>
<ul>
    {{#each site.pages.subdir.subsubdir}}
        <li>{{ url }}</li>
    {{/each}}
</ul>

<h1>Data</h1>
<p>Foo: {{ site.data.someData.foo }}</p>
<ul>
    {{#each site.data.someData.baz }}
        <li>Qux: {{ qux }}</li>
    {{/each}}
</ul>