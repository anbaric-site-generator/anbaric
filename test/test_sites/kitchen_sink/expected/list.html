<html>
  <head></head>
  <body>
    <h1>Top-level</h1>
    <ul>
      <li>/index.html</li>
      <li>/list-two.html</li>
      <li>/page2.html</li>
    </ul>
    <h1>Subdirectory</h1>
    <ul>
      <li>/subdir/page1.html</li>
      <li>/subdir/page2.html</li>
      <li>/subdir/page3.html</li>
    </ul>
    <h1>Sub-subdirectory</h1>
    <ul>
      <li>/subdir/subsubdir/foo.html</li>
    </ul>
    <h1>Data</h1>
    <p>Foo: bar</p>
    <ul>
      <li>Qux: 1</li>
      <li>Qux: 2</li>
    </ul>
    <ul>
      <li>Hello, world</li>
      <li>Good day, universe</li>
    </ul>
  </body>
</html>